pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -609,8 +619,14 @@ pub mod pallet {
 			Self::do_transfer_avatar(&seller, &buyer, &avatar.season_id, &avatar_id)?;
 			Trade::<T>::remove(avatar.season_id, avatar_id);
 
-			Accounts::<T>::mutate(&buyer, |account| account.stats.trade.bought.saturating_inc());
-			Accounts::<T>::mutate(&seller, |account| account.stats.trade.sold.saturating_inc());
+			let current_season_id = CurrentSeasonStatus::<T>::get().season_id;
```

Comment:
Can you change this line https://github.com/ajuna-network/Ajuna/blob/ae79b9b52694b169dd6a7784075435ad59d7a5d8/pallets/ajuna-awesome-avatars/src/lib.rs#L601 with the following:
```rust
let (current_season_id, Season { fee, .. }) = Self::current_season_with_id()?;
```

Reply:
Done!